include/qore/intern/QoreClassIntern.h

```diff
@@ -2074,6 +2074,10 @@ class qore_class_private {
         ClassAccess access;
         const QoreMemberInfo* omi = parseFindMember(mem, qc, access);
         if (omi) {
+            // issue #3355: to handle out of order initialization properly, we need to ensure that the member is
+            // initialized before accessing its type.  We cannot rely on calling members.parseInit() here because we
```

Comment:
There is a duplicate _we_ in the comment.
```suggestion
            // initialized before accessing its type.  We cannot rely on calling members.parseInit() here because
```